spec/apiForSheetsSpec.js

```diff
@@ -264,17 +264,37 @@ describe('Api For Sheets Module', () => {
           'https://github.com/oppia/oppia/wiki/Contributing-code-to-' +
           'Oppia#setting-things-up'
         );
-        expect(github.issues.createComment).toHaveBeenCalledWith({
-          owner: pullRequestPayload.payload.repository.owner.login,
-          repo: pullRequestPayload.payload.repository.name,
-          number: pullRequestPayload.payload.pull_request.number,
-          body:
+        var oppiaAndroidLinkText = 'here';
+        var linkOppiaAndroid = oppiaAndroidLinkText.link(
+          'https://github.com/oppia/oppia-android/wiki#onboarding-instructions'
+        );
+
+        var commentBody = '';
+        if (pullRequestPayload.payload.repository.name === 'oppia-android') {
+          commentBody = 'Hi! ' +
```

Comment:
This is not the way to do this.
You need to update the repository in the request payload to be oppia-android in a different describe block. Then assert that the correct comment is being sent.

Reply:
@jameesjohn please can you review the test as i had added different describe block to set repository as oppia android.